### ISSUE TITLE
Addition of gorouter_time to the Gorouter's Access Logs has broken Top-Logs' ability to parse them.

### ISSUE THREAD

**Issue by rhall-pivotal** [dmikusa-pivotal/top-logs#6]
Due to a new field in the **access.log** files in the Gorouter, the logs from a Gorouter in PCF 2.7.x+ will not parse through top-logs. Example:

Sample Log:
```
api.system.cf08.vanguard.com - [2020-07-22T17:00:01.945714608Z] "GET /v2/service_plans/f8bee0d6-e925-483c-a2f7-b103b07573d0/service_instances?results-per-page=100 HTTP/1.1" 200 0 19api.system.cf08.customer.com - [2020-07-22T17:00:01.945714608Z] "GET /v2/service_plans/f8bee0d6-e925-483c-a2f7-b103b07573d0/service_instances?results-per-page=100 HTTP/1.1" 200 0 1951 "-" "Go-http-client/1.1" "10.184.164.122:52324" "10.184.164.35:9024" x_forwarded_for:"10.184.164.122" x_forwarded_proto:"https" vcap_request_id:"503179f4-4d22-4699-45c9-a81bd6b606ab" response_time:0.028583 gorouter_time:0.000314 app_id:"-" app_index:"-" x_cf_routererror:"-" x_b3_traceid:"540e7db67df4d9c7" x_b3_spanid:"540e7db67df4d9c7" x_b3_parentspanid:"-" b3:"540e7db67df4d9c7-540e7db67df4d9c7"
```

Running **top-logs** on this entry:
```
./top-logs-x86_64-apple-darwin -f gorouter test.log
Parse error: Error(
    Code(
        "gorouter_time:0.000314 app_id:\"-\" app_index:\"-\" x_cf_routererror:\"-\" x_b3_traceid:\"540e7db67df4d9c7\" x_b3_spanid:\"540e7db67df4d9c7\" x_b3_parentspanid:\"-\" b3:\"540e7db67df4d9c7-540e7db67df4d9c7\"",
        Tag,
    ),
) with line 'api.system.cf08.customer.com - [2020-07-22T17:00:01.945714608Z] "GET /v2/service_plans/f8bee0d6-e925-483c-a2f7-b103b07573d0/service_instances?results-per-page=100 HTTP/1.1" 200 0 1951 "-" "Go-http-client/1.1" "10.184.164.122:52324" "10.184.164.35:9024" x_forwarded_for:"10.184.164.122" x_forwarded_proto:"https" vcap_request_id:"503179f4-4d22-4699-45c9-a81bd6b606ab" response_time:0.028583 gorouter_time:0.000314 app_id:"-" app_index:"-" x_cf_routererror:"-" x_b3_traceid:"540e7db67df4d9c7" x_b3_spanid:"540e7db67df4d9c7" x_b3_parentspanid:"-" b3:"540e7db67df4d9c7-540e7db67df4d9c7"'

Duration: 9999-12-31 23:59:59 -05:00 to 0000-01-01 00:00:00 -05:00


Total Requests: 0
Total Errors  : 1

Response Codes:

++
++

Request Methods:

++
++

Top '10' Requests (no query params)

++
++

Top '10' Requests (with query params)

++
++
```

I believe this is occurring due to the addition of the `gorouter_time` field to the Access.log entries. This was added in PCF 2.7.10 - [Release Notes for 2.7.10](https://docs.pivotal.io/platform/2-7/release-notes/runtime-rn.html#2.7.10)

This has been updated in the [About Access Logs Docs](https://docs.pivotal.io/platform/application-service/concepts/architecture/router.html#about-access-logs).

**Comment by rhall-pivotal** [comment]
To double-check, I ran **top-logs** on this older access log:

```
api.wdc-06-pcf1-system.oc.vmware.com - [2020-04-07T17:34:23.915+0000] "GET /v2/apps/af22f651-2d07-4d1f-a3eb-98245aae94d0/env HTTP/1.1" 200 0 6672 "-" "Java/1.8.0_222" "10.206.174.201:48336" "192.168.20.23:9024" x_forwarded_for:"192.168.20.93, 10.206.174.201" x_forwarded_proto:"https" vcap_request_id:"25b2bcd7-d743-43cd-4d2c-9b1399d17c1a" response_time:0.054948149 app_id:"-" app_index:"-" x_b3_traceid:"49b97d9d23737c58" x_b3_spanid:"27176c2da28055b6" x_b3_parentspanid:"49b97d9d23737c58"
```

This seems to be the problem.

**Comment by dmikusa-pivotal** [comment]
Resolved in top-logs 1.3.0+.

https://github.com/dmikusa-pivotal/top-logs/releases/tag/1.3.0
